package.json

```diff
@@ -27,7 +27,9 @@
     "build": "node bin/build",
     "coverage": "istanbul -- cover ./node_modules/.bin/nodeunit tests/unit",
     "data": "node scripts/generate-identifier-data",
-    "pretest": "jshint src && jscs src",
+    "jscs": "jscs src",
+    "jshint": "node bin/jshint src",
```

Comment:
> Anyway, I've lost any interest in JSHint's development in any way and I
> have no interest nor time to explain, obvious to me, things.
> 
> Consider my PRs closed and I'll close them later myself if no one else does
> it.

Sorry you were discouraged, I'm currently trying to sort out the PR's so people do not get discouraged in the future.

Your other PR's, marked `Needs review` I am not discounting, it was just this one I was questioning to try and understand the motive behind it.